docs/src/main/asciidoc/security-overview-concept.adoc

```diff
@@ -76,19 +75,18 @@ For more information, see the link:{vault-guide}[Quarkus and HashiCorp Vault] do
 === Secure serialization
 
 If your Quarkus Security architecture includes RESTEasy Reactive and Jackson, Quarkus can limit the fields that are included in JSON serialization based on the configured security.
-For more information, see xref:resteasy-reactive.adoc#secure-serialization[Writing REST services with RESTEasy Reactive].
```

Comment:
Why is `.adoc` dropped here? Other places keep the file name suffix

Reply:
Fixed.